Template/Tag/ThemeColorTag.php

```diff
@@ -22,9 +23,9 @@ public function getIcon() {
     public function getParameters() {
         return array(
             $this->makeSetting('themeColor', '', FieldConfig::TYPE_STRING, function (FieldConfig $field) {
-                $field->title = 'Theme Color';
+                $field->title = Piwik::translate('TagManager_ThemeColorTagThemeColorTitle');
```

Comment:
It looks like `TagManager_ThemeColorTagName` already exists and has the same value.